### ISSUE TITLE
Can't load Godot 4

### ISSUE THREAD

**Issue by unsignedFoo** [godotengine/godot#74396]
### Godot version

4.0 stable Standard & Mono

### System information

Micro-Star International Co., Ltd. GL72MVR 7RFX, Windows 10, Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz, GTX 1060M, 16 GB RAM

### Issue description

Can't run Godot4, both Standard and Mono version.
Console stay at blank.
![image](https://user-images.githubusercontent.com/8471486/222930231-500fd172-7b58-43c6-8c4e-dc678b82697e.png)

### Steps to reproduce

Execute Godot executable.
Tried with both Mono and Standard

### Minimal reproduction project

N/A

**Comment by MatiasVME** [comment]
You try open with another console, like a cmd or another?

**Comment by unsignedFoo** [comment]
@MatiasVME Yes, also tried to downgrade drivers. Now NVIDIA drivers 516.94, same results.

**Comment by unsignedFoo** [comment]
Beta 6 works, drivers 516.94
![image](https://user-images.githubusercontent.com/8471486/222932938-ddba0b3d-1678-4971-a57c-3f375cdf8a02.png)

**Comment by akien-mga** [comment]
Can you try other intermediate versions between 4.0-beta6 and 4.0-stable to find the first one which stops working?

**Comment by unsignedFoo** [comment]
@akien-mga Beta 6 and Beta 7 work. From Beta 8 (included) it stops working. Runs project selector as OpenGL, when project is selected does switch to Vulkan
![image](https://user-images.githubusercontent.com/8471486/222976195-3543cc0f-fdef-492d-96a1-f79267a69967.png)
![image](https://user-images.githubusercontent.com/8471486/222976230-b380ce79-26be-498d-a480-364be012391d.png)

**Comment by akien-mga** [comment]
This may be fixed by #75881. This fix should be available in 4.0.3 or 4.1 dev 1, please test when those are released.
